test/e2e/instrumentation/logging/generic_soak.go

```diff
@@ -90,7 +90,7 @@ func RunLogPodsWithSleepOf(f *framework.Framework, sleep time.Duration, podname
 			return v1.PodSpec{
 				Containers: []v1.Container{{
 					Name:  "logging-soak",
-					Image: imageutils.GetE2EImage(imageutils.BusyBox),
+					Image: imageutils.GetE2EImage(imageutils.Agnhost),
 					Args: []string{
```

Comment:
Default Agnhost command is ``/agnhost``, so we need to override that with something like ``Command: []string{"/bin/sh"},``